lib/crud_responder/locales/en.yml

```diff
@@ -0,0 +1,12 @@
+en:
```

Comment:
The gem already has locales `cd /tmp && git clone git@github.com:olegantonyan/crud_responder.git && ls crud_responder/config/locales`